app/views/moderations/mod.html.erb

```diff
@@ -1,5 +1,7 @@
 <% title "[Moderate] " + @moderatable.title %>
 
+<%= javascript_packs_with_chunks_tag "modPage", defer: true %>
```

Comment:
Could we call this `commentModPage` instead? We have other moderation pages we still use, so IMO it's better to be specific.